web/layout_attributes/using_layout_attributes/my_horizontal_element.html

```diff
@@ -0,0 +1,12 @@
+<link rel="import" href="packages/polymer/polymer.html">
+<dom-module id="my-horizontal-element">
+  <template>
+    <section class="layout horizontal">
```

Comment:
Do these still work without any extra imports? I haven't tried them but I think the layout styles are opt-in now.